config/azure/build.yaml

```diff
@@ -55,10 +55,17 @@ jobs:
    # Provides $(Gaffer.*) variables
    - script: |
        ./config/azure/setBuildVars.py
+       echo "##vso[task.setvariable variable=Gaffer.Sphinx]"`which sphinx-build`
      displayName: 'Set Custom Variables'
      env:
        GITHUB_ACCESS_TOKEN: $(githubAccessToken)
 
+   # Docs builds regularly hang on mac, temporarily disable them for non-release build
+   - script: |
+       echo "##vso[task.setvariable variable=Gaffer.Sphinx]/dev/null"
```

Comment:
Can you explain why we are setting to `/dev/null`?

Reply:
An empty var doesn't work, but have updated the string [here](https://github.com/GafferHQ/gaffer/commit/1f55c9b14fa7911e47b9c394dbceb2b252bb4f37)